core/src/io/anuke/mindustry/entities/type/base/Hori.java

```diff
@@ -0,0 +1,4 @@
+package io.anuke.mindustry.entities.type.base;
+
+public class Hori extends MinerDrone{
+}

```

Comment:
While it doesn't do anything special you might as well use `MinerDrone` instead of an empty `Hori` child